packages/react-sdk/src/components/BoardBar/ExportWhiteboardDialogDownloadPdf.test.tsx

```diff
@@ -61,8 +76,10 @@ describe('<ExportWhiteboardDialogDownloadPdf />', () => {
       );
     };
 
-    jest.mocked(URL.createObjectURL).mockReturnValue('blob:url');
-    jest.mocked(createWhiteboardPdf).mockReturnValue(of(new Blob(['value'])));
+    vi.mocked(URL.createObjectURL).mockReturnValue('blob:url');
+    vi.spyOn(pdf, 'createWhiteboardPdf').mockReturnValue(
```

Comment:
Should be mocked because of https://github.com/nordeck/matrix-neoboard/pull/554/files#diff-c32f5e1a69cf35d2142eee95bbc066fb2c9093ade222332052580c29b0bdee7fR46

```suggestion
    vi.mocked(pdf, 'createWhiteboardPdf').mockReturnValue(
```

Reply:
This test is failing when doing this change as the pdf.createWhiteboardPdf is called 11 times instead of once. Might need cleanup to work between runs. Will recheck this at the end of me going through this.

Treat this comment as mental buffer :)